.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: ci
+on:
+  push:
+    branches:
+      - "master"
+      - "development"
+    paths-ignore:
```

Comment:
That is a really good catch, we should add this to our existing workflows in the other repos, thanks!